### ISSUE TITLE
Exported image column values don't match exported file structure

### ISSUE THREAD

**Issue by QwertyZW** [akeneo/pim-community-dev#6299]
**I don't know if this is a bug or a misconfiguration on my end**

I've created a variant group `the_variant_group` and attributed an attribute set that contains an 'image_large' attribute to it, I've also successfully assigned an image to this attribute 

The products belonging to the variant group correctly fetch the large image from the attribute set of the variant group and display it.

However when exporting the products, I get a column that looks like something like this

```
sku;image_large
123abc;files/123abc/image_large/image.jpg
```

But I get a `files` structure that looks like this

```
files/the_variant_group/image_large/image.jpg
```

and so when I import the products with Pimgento2, the images end up not being imported correctly, however if I manually massage the csv and change the column `image_large` values to match the paths, Pimgento2, successfully imports the simple product images, after having been configured to map the column `image_large` to the corresponding magento image roles through the provided pimgento2 configuration panel via the magento2 backend.

**Comment by momoss** [comment]
Hello @QwertyZW,

Thank you for raising this issue.

Unfortunately, we did not manage to reproduce it on our side. Could you provide us with more information about the PIM version you use.

Regards

**Comment by QwertyZW** [comment]
Version: 1.7.5

My appologies for missing this detail.

**Comment by solivier** [comment]
Hi  @QwertyZW,

We did not manage to reproduce your problem, even on the same PIM version than you.
Have you done any custom code that could change the export behavior, maybe you should also try to do an export without the Pimgento2 to be sure it's not due to it.

Cheers

**Comment by QwertyZW** [comment]
@solivier After some investigation I figured out that it might have been a lack of understanding of akeneo on my end and the generated file structure and the corresponding csv do make sense although they're not what I expected them to be (They're as mentioned in the OP)

TLDR the behaviour described in the OP is expected/ as designed and this ticket can be closed
